spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/SpringCloudKubernetesInformerFactoryProcessor.java

```diff
@@ -43,7 +43,7 @@
 /**
  * @author Ryan Baxter
  */
-class SpringCloudKubernetesInformerFactoryProcessor extends KubernetesInformerFactoryProcessor {
+public class SpringCloudKubernetesInformerFactoryProcessor extends KubernetesInformerFactoryProcessor {
```

Comment:
since this is re-used, it needs to be public. Actually, this one might disappear in one of the major releases, because `KubernetesInformerFactoryProcessor` is deprecated (among other deprecations) and in the nearest future I will taking a closer look into it. It is on my TODO list.